pkg/templates/imagepullpolicy/template.go

```diff
@@ -0,0 +1,39 @@
+package livenessprobe
```

Comment:
Could you write unit tests for this? Refer #199 for an example.

Reply:
sure!